components/application-mgt/org.wso2.carbon.identity.application.common/src/main/java/org/wso2/carbon/identity/application/common/model/ApplicationBasicInfo.java

```diff
@@ -49,82 +50,150 @@ public int getApplicationId() {
     /**
      * Set application id.
      *
-     * @param applicationId application id
+     * @param applicationId application id.
```

Comment:
>      * @param applicationId		Application id.

Reply:
Resolved in https://github.com/wso2/carbon-identity-framework/pull/4432